portfolio/src/main/webapp/script.js

```diff
@@ -57,6 +57,9 @@ window.onload = init;
 function init() {
     // Simulate scroll event to position elements right
     OnWindowScrolled();
+
+    // Load stuff dynamically
```

Comment:
Instead of 'stuff' it would be better to explain what it loads (but without implementation details, i.e. no specific functions, classes, etc, just what it loads in general, but more specific than 'stuff').